solutions/ha-nva/layer-7-ingress/azuredeploy.json

```diff
@@ -0,0 +1,801 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The Azure Region to Use"
+            }
+        },
+        "AvailabilityZoneCount": {
```

Comment:
I've used lower camel case for parameter names in the templates I've built. Do the same here for consistency?

AvailabilityZoneCount > availabilityZoneCount

Reply:
I think I have this scrubbed to match the convention in both the json and the table.